compose/compose_strategy.go

```diff
@@ -36,11 +36,16 @@ type CommonStrategy struct {
 	jwt.JWTStrategy
 }
 
-func NewOAuth2HMACStrategy(config *Config, secret []byte, rotatedSecrets [][]byte) *oauth2.HMACSHAStrategy {
+func NewOAuth2HMACStrategy(config *Config, secret []byte, rotatedSecrets [][]byte, optionals ...interface{}) *oauth2.HMACSHAStrategy {
```

Comment:
Please add a field to `Config` instead which specifies the entropy.

Reply:
Roger that.